desktop-cosmic/pop-gtk-theme/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=pop-gtk-theme
+PKGSEC=COSMIC
+PKGDEP="gnome-themes-standard gtk-3 murrine"
+BUILDDEP="sassc"
+PKGDES="Pop gtk theme"
```

Comment:
GTK theme found in Pop!_OS

Or something similar

Reply:
Gtk and sound themes from System76 Pop!_OS

Reply:
Sure, but it's GTK.